solana/pyth2wormhole/program/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(incomplete_features)]
```

Comment:
I think this is necessary for  the `adt_const_params` just below it in newer nightlies. Nice!